### PR TITLE
Closes #786: Migration for D7 marquee paragraph text (to text on media).

### LINKED ISSUE #786: Migrate Marquee paragraph type
See migration mapping doc for details: https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0

### PR DESCRIPTION
**Pull request by smitahantzios**
## Description
Migration for Paragraph Content Marquee to AZ Text Media

Requirements on lines 145-158 here: https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0

## Related Issue

Closes #786 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php [hunk 1]
```diff
@@ -49,6 +49,53 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
...
+        'bg-trans-sky' => 'light',
+        'bg-trans-arizona-blue' => 'dark',
+        'bg-trans-black' => 'dark',
+      ];
+      foreach ($bg_text_color as $bg_text_color_item) {
+        $behavior['bg_color'] = $bg_text_color_mapping[$bg_text_color_item['value']];
+      }
+    }
+
+    // Text Background Color.
```
Comment (danahertzberg) [inline modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php:68]: Update comments

**Comment by danahertzberg** [comment]
Still need mapping for the `Full-width background (content bottom)` view mode to change the content style to `Bottom style`. (Line 155 on the mapping spreadsheet)

I also added one more requirement in lines 156-157. If the `Default` view mode is selected in QS1, then the content style changes to `Bottom style` **and** the full width checkbox gets `unchecked`.

**Comment by smitahantzios** [comment]
> Still need mapping for the `Full-width background (content bottom)` view mode to change the content style to `Bottom style`. (Line 155 on the mapping spreadsheet)
> 
> I also added one more requirement in lines 156-157. If the `Default` view mode is selected in QS1, then the content style changes to `Bottom style` **and** the full width checkbox gets `unchecked`.

Done.

File: modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php (hunk 1 above)
Comment (smitahantzios) [reply modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php:68]: done

Comment (joeparsons) [reply modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php:68]: Looks like these comments still haven't been updated.  I think we might have been getting mixed up between this one and the full-width background one...

Comment (danahertzberg) [reply modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php:68]: I'm not seeing where this is done.. This line still says `// Text Background Color.` which is incorrect.

**Review by tadean — APPROVED** [review]

**Review by camikazegreen — APPROVED** [review]

**Pull request merged**